packages/run-protocol/src/vaultFactory/addAssetToVault.js

```diff
@@ -25,18 +31,22 @@ export const addInterchainAsset = async (
   },
   {
     options: {
-      denom,
-      keyword = 'IbcATOM',
-      proposedName = 'ATOM',
-      decimalPlaces = 4,
-    },
+      interchainOracle = 'ATOM',
```

Comment:
managing the scope of these option names seems a bit tricky. Do we document the things they need to not collide with anywhere?

Reply:
I've bundled them in `options.interchainAssetOptions` instead.  That seems to be the right kind of namespacing.